src/vmm/src/vmm_config/mmds.rs

```diff
@@ -33,6 +32,37 @@ impl MmdsNetConfig {
     }
 }
 
+/// Wrapper over the MMDS configuration.
+#[derive(Debug, Default, Deserialize, PartialEq, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct MmdsConfigWrapper {
```

Comment:
Why do we need this wrapper? Maybe explain it in a comment.

Reply:
Not used anymore.